x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -44,6 +57,22 @@ export const SessionView = ({ sessionEntityId, height, jumpToEvent }: SessionVie
   const [searchQuery, setSearchQuery] = useState('');
   const [searchResults, setSearchResults] = useState<Process[] | null>(null);
 
+  const [isOptionDropdownOpen, setOptionDropdownOpen] = useState(false)
+
+  const optionsList: optionsField[]  = [
```

Comment:
perf nit: declare it outside the Component, or this object will be declared on every re-render without needing to